src/main/java/com/example/demo/controllers/FilesUploadController.java

```diff
@@ -0,0 +1,71 @@
+package com.example.demo.controllers;
+
+import com.example.demo.service.FileUploadService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.core.io.InputStreamResource;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.web.multipart.MultipartFile;
+
+import java.io.IOException;
+import java.util.List;
+
+@RestController
+public class FilesUploadController {
```

Comment:
Please, apply auto-formatting here.
Use `@AllArgsConstructor` wherever you need a constructor